### ISSUE TITLE
po (preorders) should be renamed to PreOrder

### ISSUE THREAD

**Issue by Chgrau** [UniMath/UniMath#846]
As discussed in #842 `po` is a bad name for preorders. The other relations defined in Sets has descriptive names like `PartialOrder` or `PreorderedSet`. If we use the same naming convention the other relations use, it should be `PreOrder`.

**Comment by DanGrayson** [comment]
I agree, but because "preorder" is a word and "pre" is not, it should be `Preorder`.

**Comment by Chgrau** [comment]
Yes that is true. I suggest we change it after #843 and #842 has been resolved.

**Comment by arnoudvanderleer** [comment]
@DanGrayson Has any progress on this been made in the last 7 years? (I suppose it should be `preorder` now, instead of `Preorder`)
